### PR TITLE
StakeAngle Pledge

### PR DESCRIPTION
**Pull request by Cryptoperchik**

### CODE REVIEW

File: StakeAngle/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Agoric Validator Pledge
+
+I, StakeAngle hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.:
+
+1. The hardware my node will run meets the requirements detailed in the [Agoric Validator Runbooks](https://github.com/Agoric/agoric-sdk/wiki/Runbook%...)
```
Comment (dckc) [inline StakeAngle/README.md:5]:
```suggestion
1. The hardware my node will run meets the requirements detailed in the [Agoric Validator Runbooks](https://github.com/Agoric/agoric-sdk/wiki/Runbooks)
```

File: StakeAngle/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+2. My node will have an uptime of at least 95%
+
+3. I will set a reasonable commission rate between 3% to 10%
+
+4. I commit to not exceed 10% voting power
+
+5. I will actively participate in security and upgrade governance proposals found on [Commonwealth.im](https://commonwealth.im/agoric)
+
+6. I will report bugs I discover through the process described in the [Agoric Reporting a Security Bug Runbook](https://github.com/Agoric/agoric-sdk/wiki/Runbook%...)
```
Comment (dckc) [inline StakeAngle/README.md:15]:
```suggestion
6. I will report bugs I discover through the process described in the [Agoric Reporting a Security Bug Runbook](https://github.com/Agoric/agoric-sdk/wiki/Runbook%3A-Reporting-a-Security-Bug)
```

File: StakeAngle/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+7. I will follow the guidelines described in the [Agoric Security Coordination - Emergency Bug Response with Chain Validators Runbook](https://github.com/Agoric/agoric-sdk/wiki/Runbook%...) to maintain the health of the chain
```
Comment (dckc) [inline StakeAngle/README.md:17]:
```suggestion
7. I will follow the guidelines described in the [Agoric Security Coordination - Emergency Bug Response with Chain Validators Runbook](https://github.com/Agoric/agoric-sdk/wiki/Runbook%3A-Security-Coordination---Emergency-Bug-Response-with-Chain-Validators) to maintain the health of the chain
```

**Review by dckc — COMMENTED** [review]
This looks pretty good, to me. I'll leave approval to @jeetraut 

@jeetraut it looks like the markdown we generated had some incomplete links.

**Review by dckc — APPROVED** [review]

File: StakeAngle/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Agoric Validator Pledge
+
```
Comment (jeetraut) [inline StakeAngle/README.md:2]:
```suggestion
[StakeAngle Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper13pkmjguqfjyct7jf32fezqd53590l7mtg2cnwc)
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

**Pull request merged**